Canvas/TechDebt/Assignments/CKRichTextInputView.m

```diff
@@ -376,7 +376,7 @@ - (NSString *)replaceImagesWithThumbnailsInHTML:(NSString *)html
                 [self.webView stringByEvaluatingJavaScriptFromString:js];
             }
             
-            [loadingImages removeObjectForKey:activityImageTag];
+            [self->loadingImages removeObjectForKey:activityImageTag];
```

Comment:
will `self.loadingImages` work here?

Reply:
It's not declared as a property. `self.` causes a compile error.